CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0]
+
```

Comment:
this is not changelog formatting and the file is named CHANEGLOG.md not CHANGELOG.ms